spec/20-http_request_header_format.md

```diff
@@ -176,6 +176,8 @@ There are two additional options that vendors MAY follow:
 - A component that makes a deferred or delayed recording decision may communicate the priority of a recording by setting `sampled` flag to `1` for a subset of requests.
 - A component may also fall back to probability sampling and set the `sampled` flag to `1` for the subset of requests.
 
+**Note**: Communicating the recording decision downstream via the `sampled` flag obviously requires adding the `traceparent` header to outgoing requests in the first place. In cases where the incoming request did not have the `traceparent` header and the tracing tool decides not to record trace data, this still entails generating a `trace-id` and `parent-id` (that is, starting a trace), even if those id values are not associated with any actual trace data.
```

Comment:
side note: trace data is a good candidate to be added to the glossary. It is mentioned in spec couple times.

Reply:
Tracing tool, too, I think.